Features/Context/PlatformUI.php

```diff
@@ -384,6 +384,41 @@ public function openTreePath($path, $node)
     }
 
     /**
+     * Clicks a content browser node based on the root of the browser or a given node.
+     *
+     * @param   string          $text   The text of the node that is going to be clicked
+     * @param   NodeElement     $parentNode       The base node to expand from, if null defaults to the content browser root
+     * @throws  \Exception                  When not found
+     */
+    protected function openBrowserNode($text, $parentNode)
```

Comment:
What about `openFinderExplorerNode` ? Would it be closer from the actual role it has ?